script/1_0_2current_en_script/Common/00_00.txt

```diff
@@ -467,51 +467,51 @@ But you never caused any problems. You're still alive. So go, take everything ba
 <Page134>
 "――――――"
 ―――I will never be able to put into words the joy and warmth that filled my chest at this moment.
-"It's all right for you to be here" she said.
+'It's all right for you to be here' she said.
 She affirmed that, stronger than anyone else.
 <Page135>
 "Shiki. Even among the abilities that other people possess, the one you have is awfully unique. But the fact that you have it must have some sort of meaning.
-God doesn't share his power for no reason. There are times in your future when you'll need it, and that's why you have the Mystic Eyes of Death Perception. Even though I know how cruel this is―――"
+God doesn't share His power for no reason. There are times in your future when you'll need it, and that's why you have the Mystic Eyes of Death Perception. Even though I know how cruel this is―――"
```

Comment:
hmm are we going this route?

Reply:
CMOS (8.95) disagrees. no deity pronouns are capital.